picard/ui/options/renaming.py

```diff
@@ -22,11 +22,11 @@
 import sys
 from functools import partial
 from PyQt4 import QtGui
-from PyQt4.QtGui import QPalette, QColor
+from PyQt4.QtGui import QPalette
 from picard import config
 from picard.const import PICARD_URLS
 from picard.file import File
-from picard.script import ScriptParser, SyntaxError, ScriptError, UnknownFunction
+from picard.script import ScriptParser, SyntaxError, ScriptError
```

Comment:
I won't make an import cleanup PR across all of Picard because this requires a decent amount of testing because not all imports are side effect free (that is, simply removing the otherwise unused import might already run code).